policyengine/countries/uk/default_reform/reform.py

```diff
@@ -485,6 +487,18 @@ def formula(benunit, period, parameters):
             )
             return applicable_income
 
+    class meets_marriage_allowance_income_conditions(
+        baseline_variables["meets_marriage_allowance_income_conditions"]
+    ):
+        def formula(person, period, parameters):
+            if parameters(
+                period
+            ).reforms.abolish_marriage_allowance_income_condition:
+                return True
+            return baseline_variables[
```

Comment:
Could you add a comment on this? Haven't seen this structure before.

Reply:
Neutralizing a variable is simple in OpenFisca reforms, but making it true all the time doesn't have a simple function. So I've set the parameter to override the income condition here.